index.bs

```diff
@@ -7116,7 +7116,7 @@ values. Those values persist accros invocation of this algorithm.
 
 		5. Clamp <var>detector average</var> to a maximum of 1.0.
 
-		6. Let <var>envelope rate</var> be the result of <a>computing the envelope rate</a>.
+		6. Let <var>envelope rate</var> be the result of <a>computing the envelope rate</a> based on values of <var>attack</var> and <var>release</var>.
```

Comment:
Shouldn't the mention of `attack` and `release` be in the `computing the envelope rate` section?

Reply:
The changes below mention this, but referring to the attributes. The use of `attack` and `release` here indicates where these are used (to get rid of the single-use issue).